content/about.md

```diff
@@ -2,6 +2,38 @@
 title: About
 ---
 
-## About
+# About
 
-Help us to make _free communication_ possible and [contribute](https://invent.kde.org/melvo/xmpp-providers) to this project!
+XMPP (E**x**tensible **M**essaging and **P**resence **P**rotocol) is like a common language your and your chat partner's app speak to each other.
+Similar to email, there is a service provider managing your account and messages.
+That provider speaks XMPP too.
+
+But some XMPP providers support functionalities that other providers do not.
+Some support registrations via your app and some only via their websites.
+Some provide more space for sharing media than others or store them longer than others.
+The **XMPP Providers** makes it easy to find a suitable provider.
+
+All properties of the included providers have references in order to verify and maintain them.
+Therefore, it is important that providers add corresponding statements to their websites.
+
+This website uses XMPP Providers to generate its provider listing, show the detail pages for individual providers and explain how a provider could improve its category.
```

Comment:
This website uses **the** XMPP Providers **project** ? (+ link) to generate its provider listing, ~~show~~ the detail pages for each provider **,** and **to** explain how ...